astropy/visualization/lupton_rgb.py

```diff
@@ -155,20 +160,28 @@ def _convert_images_to_uint8(self, image_r, image_g, image_b):
         for c in image_rgb:
             c *= fac
             with np.errstate(invalid='ignore'):
-                c[c < 0] = 0                # individual bands can still be < 0, even if fac isn't
+                c[c < 0] = 0  # individual bands can still be < 0, even if fac isn't
 
         pixmax = self._uint8Max
-        r0, g0, b0 = image_rgb           # copies -- could work row by row to minimise memory usage
+        # copies -- could work row by row to minimise memory usage
+        r0, g0, b0 = image_rgb
 
-        with np.errstate(invalid='ignore', divide='ignore'):  # n.b. np.where can't and doesn't short-circuit
+        # n.b. np.where can't and doesn't short-circuit
+        with np.errstate(invalid='ignore', divide='ignore'):
             for i, c in enumerate(image_rgb):
-                c = np.where(r0 > g0,
-                             np.where(r0 > b0,
-                                      np.where(r0 >= pixmax, c*pixmax/r0, c),
-                                      np.where(b0 >= pixmax, c*pixmax/b0, c)),
-                             np.where(g0 > b0,
-                                      np.where(g0 >= pixmax, c*pixmax/g0, c),
-                                      np.where(b0 >= pixmax, c*pixmax/b0, c))).astype(np.uint8)
+                c = np.where(
+                    r0 > g0,
```

Comment:
To be consistent, should this be indented one more level?

Reply:
It is at the same level as the other nested `np.where` statements because it is acting at that level. So I think this is right.

Reply:
sure, but I was assuming those all should be indented too.  If you look at line 175 below, it's indented relative to `np.where`, but above on line 173 it is not.  So should lines, 173-183 all be indented one level?

Reply:
I see, It just so happens that `c = ` is 4 characters, which is what `black` does for each indent which is why it lines up  the the `np.where` the others doing have this because there is not the assignment part at the start.

If this part bothers you too much I can add some `fmt: skip` around it and fix it to something that you would prefer.

Reply:
no worries.  Thanks for the explanation.